### ISSUE TITLE
Create the Share Anthrogram + Refactor `BarGraph.vue`

### ISSUE THREAD

**Issue by arnonrdp** [globe-and-citizen/Celebrity-Fanalyzer#85]
The information we have about the shares makes it possible to create graphs.

```[tasklist]
### Tasks
- [x] Refactor BarGraph.vue 
- [ ] https://github.com/globe-and-citizen/Celebrity-Fanalyzer/issues/121
```

<img width="742" alt="Captura de Tela 2023-02-02 às 16 02 50" src="https://user-images.githubusercontent.com/26576201/216424878-2a3815d8-a763-4a03-83f5-f5b08c62444c.png">

Let's create a new anthrogram with the sharing information we have.

However, we need to worry about one detail here.

I noticed that our `BarGraph.vue` has some specific likes and dislikes information. Ideally, this component should be more generic, just receive data and generate graphs. Therefore, we will need to refactor it so that we can use it both for likes and dislikes and for sharing.

So in summary: the same bar chart component for both charts.
